src/neuroconv/tools/nwb_helpers/_configuration_models/_base_backend.py

```diff
@@ -29,6 +29,10 @@ class BackendConfiguration(BaseModel):
         )
     )
 
+    nwbfile_identifier: str = Field(
```

Comment:
Is this still needed anymore?

Reply:
I don't think so, it is nice provenance but our (edge case) prove it less useful to identify whether files require remapping. At the end, we do the full check.

Reply:
OK can you remove it in that case? If we don't need it yet, but we'll keep it in mind

Reply:
Sure.

Reply:
Should be done.